pkg/daemon/controller/controller.go

```diff
@@ -194,6 +283,7 @@ func (sc *ServiceController) reconcileService(service winsvc.Service, expected s
 	}
 
 	if updateRequired {
+		klog.Infof("updating %s service", expected.Name)
```

Comment:
```suggestion
		klog.Infof("updating service %s ", expected.Name)
```

For consistency with above logs.